setup.py

```diff
@@ -355,7 +355,7 @@ def finalize_options(self):
         "onnxruntime_pybind11_state.so",
         "libdnnl.2.dylib",
         "mimalloc.so",
-        "libonnxruntime.dylib*",
+        "libonnxruntime.*.dylib",
```

Comment:
To also cover libonnxruntime.dylib, shall we use:
```suggestion
        "libonnxruntime*.dylib",
```

Reply:
done